driver-adapters-wasm/d1-cf-basic/src/index.js

```diff
@@ -0,0 +1,190 @@
+            age: true,
+            name: true,
+          },
+        }),
+        findMany: await prisma.user.findMany({
+          select: {
+            email: true,
+            age: true,
+            name: true,
+          },
+          orderBy: {
+            email: 'asc',
+          },
+        }),
+        findUnique: await prisma.user.findUnique({
+          where: {
+            email: 'test-1@prisma.io',
+          },
+          select: {
+            email: true,
+            age: true,
+            name: true,
+          },
+        }),
+        update: await prisma.user.update({
+          where: {
+            email: 'test-1@prisma.io',
+          },
+          data: {
+            age: 26,
+          },
+          select: {
+            email: true,
+            age: true,
+            name: true,
+          },
+        }),
+        updateMany: await prisma.user.updateMany({
+          where: {
+            age: 26,
+          },
+          data: {
+            age: 27,
+          },
+        }),
+        findFirst: await prisma.user.findFirst({
+          where: {
+            age: 27,
+          },
+          select: {
+            email: true,
+            age: true,
+            name: true,
+          },
+        }),
+        delete: await prisma.user.delete({
+          where: {
+            email: 'test-1@prisma.io',
+          },
+          select: {
+            email: true,
+            age: true,
+            name: true,
+          },
+        }),
+        count: await prisma.user.count(),
+        aggregate: await prisma.user
+          .aggregate({
+            _avg: {
+              age: true,
+            },
+          })
+          .then(({ _avg }) => ({ age: Math.trunc(_avg?.age ?? 0) })),
+        groupBy: await prisma.user.groupBy({
+          by: ['age'],
+          _count: {
+            age: true,
+          },
+        }),
+        // TODO skipped because of bug: "User Not Found"
+
+        // findFirstOrThrow: await prisma.user.findFirstOrThrow({
+        //   select: {
+        //     age: true,
+        //     email: true,
+        //     name: true,
+        //   },
+        // }),
+        // findUniqueOrThrow: await prisma.user.findUniqueOrThrow({
+        //   where: {
+        //     email: 'test-2@prisma.io',
+        //   },
+        //   select: {
+        //     age: true,
+        //     email: true,
+        //     name: true,
+        //   },
+        // }),
+        // Skipping this because of too many sub-requests (limit is 50 per fetch call)
+
+        // upsert: await prisma.user.upsert({
+        //   where: {
+        //     email: 'test-4@prisma.io',
+        //   },
+        //   create: {
+        //     email: 'test-4@prisma.io',
+        //     age: 30,
+        //     name: 'Test 4',
+        //   },
+        //   update: {},
+        //   select: {
+        //     email: true,
+        //     age: true,
+        //     name: true,
+        //   },
+        // }),
+      }
```

Comment:
I guess this can be enabled since it's not an http resquest?

Reply:
I put that back, I wanted to but forgot at some point, thanks for mentioning it!